include/class_info.php

```diff
@@ -50,105 +52,97 @@ public static function import_status() {
 
 		$arrIndexed = self::number_of_entries();
 
-		if(count($arrPostCount) > 0)
-		{
-			$countIndexed = 0;
-
-			foreach($arrPostCount as $posts)
-			{
-				if($posts->pinged == "indexed" || $posts->pinged == "linksconverted")
-				{
-					$countIndexed = $posts->entryCount;
-				}
-				else
-				{
-
-					$countImported = $posts->entryCount;
-				}
-			}
+		if(count($arrPostCount) == 0)
+			return "No entries have been imported yet. <a href=\"" . $_SERVER['REQUEST_URI']  . "\">refresh page</a>";
 
-			if(!get_option("importStatus"))
-			{
-				return "The import status will display here.<br>";
-			}
+		$countIndexed = 0;
 
-			if(get_option("importStatus") == "importFinished")
+		foreach($arrPostCount as $posts)
+		{
+			if($posts->pinged == "indexed" || $posts->pinged == "linksconverted")
 			{
-				if($posts->post_date != NULL)
-				{
-					$status .= "Last import of configured xhtml was at " . $posts->post_date . " (server time).<br>";
-					$status .= "Download data sent from FLEx: ";
-
-					$subdomain = array_shift((explode('.', $_SERVER['HTTP_HOST'])));
```

Comment:
This was failing in unit tests. Replaced this line with
```php
$subdomain = explode('.', $_SERVER['HTTP_HOST'])[0];
```
because `array_shift` takes a by-ref parameter in older version of PHP.